mailpile/plugins/vcard_face.py

```diff
@@ -0,0 +1,59 @@
+import time
+import warnings
+
+from mailpile.i18n import gettext as _
+from mailpile.mailutils.safe import safe_decode_hdr
+from mailpile.plugins import PluginManager
+from mailpile.vcard import MailpileVCard, VCardImporter, VCardLine
+
+
+_plugins = PluginManager(builtin=__file__)
+
+
+class FaceImporter(VCardImporter):
+    """
+    Get avatar from Face header.
+    """
+    HOOKS = ['INCOMING_EMAIL']
+    FORMAT_NAME = 'Face'
+    FORMAT_DESCRIPTION = _('Get avatar from Face header.')
+    SHORT_NAME = 'face'
+    CONFIG_RULES = {
+        'active': [_('Enable this importer'), bool, True],
+    }
+    VCARD_TS = 'x-face-ts'
+    VCARD_IMG = ''
+
+    def __init__(self):
+        pass
+
+    def __call__(self, **kwargs):
```

Comment:
This is pretty vague. Why can this not have a proper name and named arguments? I don't think the complete lack of structure is a good code smell.